src/router/index.ts

```diff
@@ -28,6 +28,21 @@ const routes = [
         name: "courses",
         component: async () => import("@/pages/Page2.vue"),
       },
+      {
+        path: '/regulations',
+        name: "regulations.list",
+        component: async () => import("@/pages/regulations/List.vue"),
+      },
+      {
+        path: "/regulations/:slug",
+        name: "regulations.view",
+        component: async () => import("@/pages/regulations/View.vue"),
+      },
+      {
+        path: "/regulations/:slug/update",
```

Comment:
bunu update yerine edit yapalım.
yukarda bende profile için olanı edit yapmışım, onun gibi. 

![image](https://user-images.githubusercontent.com/6912824/227774159-46dc2f2b-5c1e-4a40-bf76-2262302f38f2.png)